workbench/views.py

```diff
@@ -80,14 +79,20 @@ def show_scenario(request, scenario_id, view_name='student_view', template='work
     })
 
 
-def user_list(request):
+def user_list(_request):
```

Comment:
`user_list` is actually a view defined in `workbench/urls.py`, so the `_request` object will always be passed along.